README.md

```diff
@@ -36,3 +37,9 @@ Check whether the code contains copyright
 - `includes` - directory to add copyright.
 - `--excludes` - exclude directory.
 - `--suffixes` - copyright will be added to files with suffix.
+
+### check-ecosystem-validity
+
+Check the validity of the ecosystem yaml file
+
+- `--input_files` - path of the project yaml
```

Comment:
This hook requires extra dependencies so it would be better to provide a full config like https://github.com/open-mmlab/pre-commit-hooks/blob/0672dd0fc5db50a17b544b3cf934f30f06f6b39a/README.md#check-model-index